<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimaryControls</class>
 <widget class="QWidget" name="PrimaryControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>307</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <pointsize>-1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: Roboto; font-size: 14px;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="1" column="6" rowspan="3">
    <widget class="QProgressBar" name="dmaUsage">
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="QCheckBox" name="timedRunCheckbox">
     <property name="text">
      <string>Timed run</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>DMA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="4" colspan="3">
    <widget class="QTextBrowser" name="notificationsTextArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="8" rowspan="3">
    <widget class="QProgressBar" name="FileFillStatus">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="9">
    <widget class="QLabel" name="streamStatusLabel">
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RUNNING</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="QSpinBox" name="timedRunValue">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string>ms</string>
     </property>
     <property name="minimum">
      <number>50</number>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
     <property name="singleStep">
      <number>500</number>
     </property>
    </widget>
   </item>
   <item row="3" column="9">
    <widget class="QPushButton" name="streamStartStopButton">
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-family: Roboto; font-size: 16px;</string>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>RAM</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="4" column="8">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>FILE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="5" rowspan="3">
    <widget class="QProgressBar" name="RAMFillStatus">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
